app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -197,6 +197,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 */
 		var subtitlesTextSize = floatPreference("subtitles_text_size", 1f)
 
+		/**
+		 * Subtitles offset
+		 */
+		var subtitlesOffset = floatPreference("subtitles_offset", 0.08f)
```

Comment:
There are two kinds of offsets; time offset and position offset. Although we don't currently support the first we should at least make sure we prepare for it in the preference name so it won't be confused in the future.

```suggestion
		var subtitlesOffsetPosition = floatPreference("subtitles_offset_position", 0.08f)
```